idl/github.com/uber/cadence/shared.thrift

```diff
@@ -130,6 +130,7 @@ enum DecisionTaskFailedCause {
   BAD_CANCEL_WORKFLOW_EXECUTION_ATTRIBUTES,
   BAD_REQUEST_CANCEL_EXTERNAL_WORKFLOW_EXECUTION_ATTRIBUTES,
   BAD_CONTINUE_AS_NEW_ATTRIBUTES,
+  BAD_START_TIMER_DUPLICATE_ID,
```

Comment:
BAD_ usually indicates that the request params are invalid i.e. malformed / missing fields etc. In this case, the request is a duplicate. How about renaming this to something like TIMER_ALREADY_EXIST or DUPLICATE_TIMER ?